backend/src/main/resources/data.sql

```diff
@@ -1,10 +1,10 @@
 INSERT INTO STAFF(first_name, surname, start_date, contracted_hours, hourly_rate, role, job_title,
-active)
+inactive)
 VALUES
-    ('Joe', 'Bloggs', '2018-08-03', 40, 1.23, 'USER', 'barista', true),
-    ('Mary', 'McGee', '2017-08-12', 40, 999.99, 'MANAGER', 'supervisor', true),
-    ('Json', 'Server', '2015-01-12', 40, 0.01, 'USER', 'floor', true),
-    ('Arthur', 'Guinness', '2019-12-12', 40, 2.99, 'USER', 'chef', true);
+    ('Joe', 'Bloggs', '2018-08-03', 40, 1.23, 'USER', 'barista', false),
+    ('Mary', 'McGee', '2017-08-12', 40, 999.99, 'MANAGER', 'supervisor', false),
+    ('Json', 'Server', '2015-01-12', 40, 0.01, 'USER', 'floor', false),
```

Comment:
surely this has to be Jason? 😂

Reply:
I'm guessing this is another one of @tjanowsk 's easter eggs. I'm loving them! Can't believe i missed it in his PR.
I'm sure it was specifically for @Skoomaki

Reply:
As much as I'd like to take credit, this is actually taken verbatim from
https://docs.google.com/document/d/1amxWzf-OZsJokOBTwgHSoibYUACiPXb0IEyx-cOv0SY/edit
so it's @smangan360 easter egg